run_sim.py

```diff
@@ -74,14 +85,44 @@ def run_sim(args, numHosts=None):
         pass
     os.mkdir(OUTPUT_DIR)
 
-    p = subprocess.Popen(simArgs, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    # Want to handle 3 scenarios:
+    #   1. First time connecting, which will ask "Are you sure you want to continue connecting?"
+    #   2. Just asking for password
+    #   3. No password is asked for because you have a key or something.
+    # code from http://linux.byexamples.com/archives/346/python-how-to-access-ssh-with-pexpect/
+    ssh_newkey = "Are you sure you want to continue connecting"
+    p = pexpect.spawn(simArgs[0], simArgs[1:])
+    p.logfile_read = sys.stdout # only log what the child process sends back
+    i = p.expect([ssh_newkey, ".*password:", pexpect.EOF, pexpect.TIMEOUT], timeout=2)
+    if i == 0:
+        print "\n[info] Saying yes to first time connection."
+        p.sendline("yes")
+        i = p.expect([ssh_newkey, ".*password:", pexpect.EOF, pexpect.TIMEOUT], timeout=2)
+    if i == 1:
+        print "\n[info] Providing user password."
+        p.sendline(KEY)
+        j = p.expect([".*denied.*", pexpect.EOF, pexpect.TIMEOUT], timeout=2)
+        if j == 0:
+            sys.exit("\n[ERROR] Did you fill in your password?")
+        elif j == 3:
```

Comment:
Sometimes mav-2 doesn't respond with a permission denied within this timeout window. Can look at making this a little more robust.